.circleci/config.yml

```diff
@@ -43,6 +43,33 @@ jobs:
           paths:
             - tmp/tarballs/*
             - node_modules/cli-engine/bin/version
+
+  generate_dev_center_docs:
+    <<: *defaults
+    filters:
+      tags:
```

Comment:
should be able to use this https://github.com/heroku/cli/blob/master/.circleci/config.yml#L11